src/instant.ts

```diff
@@ -59,14 +59,23 @@ export class Instant {
   }
 
   get epochSeconds(): number {
+    if (this.nanoseconds < 0 && this.nanoseconds % BigInt(BILLION) !== 0n) {
```

Comment:
fyi, this is a small tweak to how rounding works related to a change in the polypill (and spec I believe)